config.txt

```diff
@@ -0,0 +1,5 @@
+;!@Install@!UTF-8!
+Title="Red Hat Developer Platform"
+BeginPrompt="Do you want to install the Red Hat Developer Platform?"
```

Comment:
There should be no additional prompt, Pete asked to remove that after first demo.